lib/routes/data/styles.js

```diff
@@ -63,4 +63,21 @@ module.exports = {
       ],
     ],
   },
+  breadcrumb: {
+    item: [
+      {
+        title: 'Dashboard',
```

Comment:
This is sample data for the breadcrumbs. Where that data is coming from for actual implementation is out of scope for this PR